flink-streaming-connectors/flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkTest.java

```diff
@@ -105,18 +105,13 @@ private void testDownBehavior(FlinkJedisConfigBase config) throws Exception {
 		try {
 			redisSink.open(new Configuration());
 		} catch (Throwable e) {
-			Throwable init = e;
 
-			// search for nested JedisConnectionExceptions
+			// search for the JedisConnectionExceptions
 			// because this is the expected behavior
-
-			int depth = 0;
-			while (!(e instanceof JedisConnectionException)) {
-				e = e.getCause();
-				if (e == null || depth++ == 20) {
-					init.printStackTrace();
-					fail("Test Failed: " + init.getMessage());
-				}
+			
+			if (!(e instanceof JedisConnectionException)) {
+				e.printStackTrace();
+				fail("Test Failed: " + e.getMessage());
```

Comment:
I also thought so that is why I had it like this but I thought your previous comment was to remove it.
Probably i misunderstood. Actually the exception was in the first layer since the beginning.
I can change it back ;)